docs/concepts/html/elements.md

```diff
@@ -244,3 +244,110 @@ impl Component for MyComponent {
 }
 ```
 <!--END_DOCUSAURUS_CODE_TABS-->
+
+## Event Types
+
+:::note
+In the following table, "`web-sys` Event Type" should be considered if you're using `yew` with `web-sys` (default); `stdweb` if you use `yew-stdweb`. See [Choose web-sys or stdweb](https://yew.rs/docs/getting-started/choose-web-library) for more information.
```

Comment:
I don't think the semicolon really helps here (maybe two separate sentences would make it better?). It would be nice if this could be split up a little bit (as part of an ongoing attempt of mine to make diffs readable).

```suggestion
In the following table `web-sys`'s event types should only be used if you're using `yew` with `web-sys`
(this is enabled by default). Use `stdweb`'s event types if you're using the `yew-stdweb` crate. See 
[the documentation page about whether to choose `web-sys` or `stdweb`](https://yew.rs/docs/getting-started/choose-web-library) for more information.
```

Reply:
I feel like that's a lot unnecessary words and not really concise.
> as part of an ongoing attempt of mine to make diffs readable

What makes the current sentence unreadable?

Reply:
> What makes the current sentence unreadable?

It's not the actual sentence – it's just handy to cap the line lengths so that when it comes to reviewing later changes it's easy to see exactly what's been changed.

I'm not particularly concerned about the exact wording, but the current sentence has a few problems:
* "Event Type" isn't a proper noun so should be lower case. I think it's better to say something like "event types from `web-sys`" or `web-sys`'s event types rather than `web-sys` event type (there are multiple event types so I think that should be plural).
* The semicolon doesn't make gramatical sense – they can't be used to connect clauses which can't stand independent of each other (i.e. they must be complete sentences).
* "See Choose `web-sys` or `std-web`" doesn't really fit very well. "Choose" definitely has to be lower case there and I think it should be in the present participle rather than in the infinitive (i.e. "choosing" rather than "choose").

Reply:
Understandable, I have merged your suggestion.